src/Aqua.jl

```diff
@@ -105,6 +116,13 @@ function test_all(
             test_persistent_tasks(testtarget; askwargs(persistent_tasks)...)
         end
     end
+    @testset "Undocumented names" begin
+        if undocumented_names !== false
```

Comment:
It looks to me that this Codecov warning is not accurate, right?

Reply:
indeed, Codecov misses the lines that start with `@testset`